package taskly.system.exception

/**
 * @author Alexandru Stoica
 * @version 1.0
 */

class UserNotFound: RuntimeException("404 User Not Found")
